website/blog/2021-12-16-lakehouse-concurrency-control-are-we-too-optimistic.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Lakehouse Concurrency Control: Are we too optimistic?"
+excerpt: "Vinoth Chandar, Creator of Apache Hudi, dives into concurrency control mechanisms"
+author: Vinoth Chandar
```

Comment:
I think it has to be "vinoth" to link to my user profile link?

Reply:
good catch, fixed